src/geocoder.js

```diff
@@ -28,14 +37,18 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
```

Comment:
See: http://googlegeodevelopers.blogspot.de/2016/06/building-for-scale-updates-to-google.html

> As of June 22, 2016 we are making the following changes to the Google Maps APIs Standard Plan:
> We no longer support keyless access (any request that doesn't include an API key). Future product updates are only available for requests made with an API key. API keys allow us to contact developers when required and help us identify misbehaving implementations.